yarn.lock

```diff
@@ -9,6 +9,13 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
+"@babel/code-frame@^7.8.3":
+  version "7.8.3"
+  resolved "https://artifactory.klarna.net/artifactory/api/npm/v-npm-production/@babel/code-frame/-/code-frame-7.8.3.tgz#33e25903d7481181534e12ec0a25f16b6fcf419e"
```

Comment:
Looks like you have references to Klarna's NPM registry.

Reply:
Classic

Reply:
AHHHAEHAEHAEHHEA damn